scripts/lib/CIME/build.py

```diff
@@ -215,7 +215,7 @@ def _build_libraries(case, exeroot, sharedpath, caseroot, cimeroot, libroot, lid
         if (not os.path.exists(shared_item)):
             os.makedirs(shared_item)
     mpilib = case.get_value("MPILIB")
-    libs = ["gptl", comp_interface, "pio", "csm_share"]
+    libs = ["gptl", "mct", "pio", "csm_share"]
```

Comment:
If we don't build nuopc here, when does it get built?

Reply:
This is the external mct library which we still need for both nuopc and moab - eventually we will remove mct from this list when building nuopc, but not ready for that yet.